apps/ledger-live-desktop/src/renderer/components/PostOnboardingHub/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useCallback } from "react";
+import { Flex, Text, Box } from "@ledgerhq/react-ui";
+import { useHistory } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { useTranslation } from "react-i18next";
+import {
+  useAllPostOnboardingActionsCompleted,
+  usePostOnboardingHubState,
+} from "@ledgerhq/live-common/postOnboarding/hooks/index";
+import {
+  clearPostOnboardingLastActionCompleted,
+  setPostOnboardingActionCompleted,
+} from "@ledgerhq/live-common/postOnboarding/actions";
+
+import PostOnboardingActionRow from "./PostOnboardingActionRow";
+
+const PostOnboardingHub = () => {
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  const { lastActionCompleted, actionsState } = usePostOnboardingHubState();
+  const { actionCompletedHubTitle, actionCompletedPopupLabel } = lastActionCompleted ?? {};
+
+  const clearLastActionCompleted = useCallback(() => {
```

Comment:
The idea is to call this function when leaving the post onboarding hub, so for that you could use the cleanup function of a `useEffect` that has no dependencies (that function will be called when the component gets unmounted)